components/menu/menu-item-mixin.js

```diff
@@ -18,6 +18,7 @@ export const MenuItemMixin = superclass => class extends superclass {
 			/**
 			 * @ignore
 			 */
+			// eslint-disable-next-line lit/no-native-attributes
 			hidden: { type: Boolean, reflect: true },
```

Comment:
Ignoring this one for now because there's logic wired up in `updated`. That should probably switch to a `MutationObserver`.